src/services/concretes/node.service.ts

```diff
@@ -0,0 +1,44 @@
+import { logger } from "@blockr/blockr-logger";
+import { inject, injectable } from "inversify";
+import { BlockJob } from "../../jobs/concretes/block.job";
+import { BlockchainInitializationService } from "./blockchainInitialization.service";
+
+@injectable()
+export class NodeService {
+    private readonly blockchainInitializationService: BlockchainInitializationService;
+    private readonly blockJob: BlockJob;
+
+    constructor(@inject(BlockchainInitializationService)
+                    blockchainInitializationService: BlockchainInitializationService,
+                @inject(BlockJob) blockJob: BlockJob) {
+        this.blockchainInitializationService = blockchainInitializationService;
+        this.blockJob = blockJob;
+    }
+
+    public async start(): Promise<void> {
+        return new Promise(async (resolve, reject) => {
+            logger.info(`${this.constructor.name} is starting.`);
+
+            try {
+                // get validator --> als undefined ben ik de eerste
```

Comment:
Dutch comment in the code
```suggestion
                // get validator --> if undefined I am the first
```